src/main/java/com/dreamsportslabs/guardian/dto/response/ScopeResponseDto.java

```diff
@@ -0,0 +1,19 @@
+package com.dreamsportslabs.guardian.dto.response;
+
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
Why do we need so many ways to create this object?
Also its an anti pattern to have setters and Builder pattern both.
Re think the object creation and modification aspect.